### PR TITLE
Add welcome message on adhesion form

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/Form/UpdateMembershipRequestType.php [hunk 1]
```diff
@@ -16,17 +16,21 @@ class UpdateMembershipRequestType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        if (!$options['is_adhesion_form']) {
```
Comment (HeahDude) [inline src/Form/UpdateMembershipRequestType.php]: What about a `UpdateUserType` and another that extends it to add this three fields?

Comment (B-Galati) [reply src/Form/UpdateMembershipRequestType.php]: Yeap let's do this. I will create a join type.

File: src/Controller/EnMarche/MembershipController.php [hunk 2]
```diff
@@ -79,8 +79,9 @@ public function joinAction(Request $request, EntityManagerInterface $manager): R
             throw $this->createNotFoundException();
         }
 
+        $fromActivation = $request->query->getBoolean('from_activation');
         $membership = MembershipRequest::createFromAdherent($user);
-        $form = $this->createForm(UpdateMembershipRequestType::class, $membership)
+        $form = $this->createForm(UpdateMembershipRequestType::class, $membership, ['is_adhesion_form' => true])
```
[the remark this answers is not in the document]
Comment (B-Galati) [reply src/Controller/EnMarche/MembershipController.php]: Yea this is wrong >-<

File: src/Form/UpdateMembershipRequestType.php (hunk 1 above)
Comment (B-Galati) [reply src/Form/UpdateMembershipRequestType.php]: done

File: src/Form/BecomeAdherentType.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class BecomeAdherentType extends AbstractType
```
Comment (HeahDude) [inline src/Form/BecomeAdherentType.php]: A membership is about becoming adherent. Please keep this one as `MembershipRequestType`.

File: src/Form/BecomeAdherentType.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            ->remove('firstName')
+            ->remove('lastName')
+            ->remove('emailAddress')
+            ->remove('position')
+        ;
+    }
+
+    public function getParent()
+    {
+        return MembershipRequestType::class;
```
Comment (HeahDude) [inline src/Form/BecomeAdherentType.php]: Please rename this one instead, something like `UserProfileType` or `User...whatever`

Comment (B-Galati) [reply src/Form/BecomeAdherentType.php]: Ok but it makes more sense to call it `AdherentType` as it's about modifying an `Adherent`.

File: src/Form/BecomeAdherentType.php (hunk 3 above)
Comment (B-Galati) [reply src/Form/BecomeAdherentType.php]: Ok

Comment (B-Galati) [reply src/Form/BecomeAdherentType.php]: Just did it and I don't like it. It's more confusing IMO.
I understand your point but from an external point of view this name makes more sense.
It is really obvious to understand and it's the main goal here no ?
You can become an adherent with this form, that's it.
What do you think ?

Comment (HeahDude) [reply src/Form/BecomeAdherentType.php]: `AdherentType` <3. Thanks!

File: src/Form/RegistrationType.php [hunk 5]
```diff
@@ -12,7 +12,7 @@
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class NewMemberShipRequestType extends AbstractType
+class RegistrationType extends AbstractType
```
Comment (HeahDude) [inline src/Form/RegistrationType.php]: `UserRegistrationType`?

**Pull request merged**